src/selectors/getRelativeSources.js

```diff
@@ -31,10 +31,9 @@ function formatSource(source: Source, root): RelativeSource {
 export function getRelativeSources(state: State): RelativeSource[] {
   const sources = getSources(state);
   const root = getProjectDirectoryRoot(state);
-
   return sources
     .valueSeq()
     .toJS()
-    .filter(({ url }) => url && url.includes(root))
+    .filter(({ url }) => url && url.startsWith(root))
```

Comment:
This change, from `includes` to `startsWith`, is causing no sources to display when a root is set.  Do we need this?

Reply:
i think startsWith should now work because of the other changes